### DISCUSSION TITLE
Publishing to a subscription outside of TypeGraphQL

### DISCUSSION THREAD

**Discussion by jimbuck** [MichalLytek/type-graphql#994] [Q&A] [answered]
I want to trigger a subscription from outside a Resolver class (inside of an Express handler). I checked the source but couldn't find anything that hinted at a way to create a `@PubSub()` instance. I am using DI within the Express handler, so is there something I can inject to emit Publisher events? Thanks!

**Comment by MichalLytek** [comment, marked as answer]
You can use custom pub sub instance:
https://typegraphql.com/docs/subscriptions.html#using-a-custom-pubsub-system

And put that in container or just export and import directly.

**Comment by akash-mahmud** [comment]
I am using express js and type-graphql, but the example implementation doesn't work for me. 
Any suggestions?
